### ISSUE TITLE
Adding or Removing Catalog Items

### ISSUE THREAD

**Issue by muralitharanvellaisamy** [cvdlab/react-planner#181]
Hi Team,
I need to add some elements to the Catalog and also need to remove some items from Catalog List. What is the method to achieve this. Thanks.

**Comment by ShubhamTyagiZed** [comment]
I need to so the same. Did you find any solution?

**Comment by alessiocarrafa** [comment]
Did you read the documentation?

**Comment by ShubhamTyagiZed** [comment]
Yes, I am now able to add my elements in the catalog but they are not
visible in the 3D view.
I am using .obj and .mlt file just like it is used in "sofa".
I have changes all the references to the files like .mtl, .obj, texture,
and image. Basically what is did is duplicate the sofa element and changes
all the ref in the planner-element.jsx.
Do I need to change anything else?


Thanks for your quick reply. I really appreciate it.

**Comment by alessiocarrafa** [comment]
Wich program ( and version ) did you use for generate it? Did the developer console print errors/infos/logs?

**Comment by ShubhamTyagiZed** [comment]
I am using Visual Studio Code and Google chrome.
I am actually new to React and Three.
Do I have change any value or multiplier in the planner-emement.jsx?
